packages/autocomplete-js/src/autocomplete.ts

```diff
@@ -150,13 +154,15 @@ export function autocomplete<TItem extends BaseItem>(
       components: props.value.renderer.components,
       container: props.value.renderer.container,
       createElement: props.value.renderer.renderer.createElement,
+      html,
       dom: dom.value,
       Fragment: props.value.renderer.renderer.Fragment,
       panelContainer: isDetached.value
         ? dom.value.detachedContainer
         : props.value.renderer.panelContainer,
       propGetters,
       state: lastStateRef.current,
+      renderer: props.value.renderer,
```

Comment:
We already pass `props.value.renderer.renderer.createElement` above so we can pass `props.value.renderer.renderer.render` here.

Reply:
Or we refactor to only pass the full renderer and write a bit less code (see a94aeecc), wdyt?